python/cuml/cuml/ensemble/randomforestclassifier.py

```diff
@@ -33,6 +33,11 @@ class RandomForestClassifier(BaseRandomForestModel, ClassifierMixin):
       https://docs.rapids.ai/api/cuml/nightly/pickling_cuml_models.html
       for more details.
 
+    .. note:: When converting to sklearn using `as_sklearn()`, the
+      `feature_importances_` attribute will return NaN values. If you need
+      feature importances, save them before conversion:
+      `importances = cuml_model.feature_importances_`
```

Comment:
We should document `feature_importances_` in the `Attributes` section at the bottom of both `RandomForestClassifier` and `RandomForestRegressor`. Can you also move this note down to the documented attribute rather than highlighting up here? I think that location makes more sense.